test/unit/org/apache/cassandra/config/CassandraRelevantPropertiesTest.java

```diff
@@ -88,92 +79,65 @@ public void testBoolean()
     @Test
     public void testBoolean_null()
     {
-        try
+        try (WithProperties properties = new WithProperties())
         {
             TEST_CASSANDRA_RELEVANT_PROPERTIES.getBoolean();
```

Comment:
btw. there is missing assertion in this test case

Reply:
Will add it.